tests/features/custom_page/navigation_menu.feature

```diff
@@ -245,3 +245,112 @@ Feature: Navigation menu for custom pages
+  Scenario: Only custom page entries can be nested in the collection navigation menu.
+    Given the following collection:
+      | title | Ergonomic backpacks |
+      | state | validated           |
+    And custom_page content:
+      | title              | collection          | status    |
+      | Types of backpacks | Ergonomic backpacks | published |
+      | Frameless          | Ergonomic backpacks | published |
+      | External frame     | Ergonomic backpacks | published |
+      | Internal frame     | Ergonomic backpacks | published |
+      | Bodypack           | Ergonomic backpacks | published |
+    And the following collection menu structure:
+      | title              | parent             | weight |
+      | Types of backpacks |                    | 3      |
+      | Frameless          | Types of backpacks | 4      |
+      | External frame     | Types of backpacks | 5      |
+      | Internal frame     |                    | 6      |
+      # Force a reserved page to be nested. This is not possible through the UI.
+      | About              | Bodypack           | 7      |
+
+    When I am logged in as a facilitator of the "Ergonomic backpacks" collection
+    And I go to the "Ergonomic backpacks" collection
+    And I click the contextual link "Edit menu" in the "Left sidebar" region
+    # The "About" page has been moved back to first level.
+    Then the menu table should be:
+      | title              | parent             |
+      | Overview           |                    |
+      | Members            |                    |
+      | Bodypack           |                    |
+      | About              |                    |
+      | Types of backpacks |                    |
+      | Frameless          | Types of backpacks |
+      | External frame     | Types of backpacks |
+      | Internal frame     |                    |
+    When I drag the "External frame" table row to the left
+    Then the menu table should be:
+      | title              | parent             |
+      | Overview           |                    |
+      | Members            |                    |
+      | Bodypack           |                    |
+      | About              |                    |
+      | Types of backpacks |                    |
+      | Frameless          | Types of backpacks |
+      | External frame     |                    |
+      | Internal frame     |                    |
+    When I drag the "Internal frame" table row to the right
+    Then the menu table should be:
+      | title              | parent             |
+      | Overview           |                    |
+      | Members            |                    |
+      | Bodypack           |                    |
+      | About              |                    |
+      | Types of backpacks |                    |
+      | Frameless          | Types of backpacks |
+      | External frame     |                    |
+      | Internal frame     | External frame     |
```

Comment:
Sorry, I am not jumping into the QA, but this ticket is blocking mine so I had to get the code from it. Normally the test would fail at this point but this piece over here does not ensure the maximum indentation as the child cannot go further than one level down the parent. It would take a second child element to be nested further down. I suggest you ignore this one as I am going to fix it in ISAICP-4609.

Reply:
I didn't get what you are trying to explain 😕